### PR TITLE
Upgrade dependencies to match quality checklist

### LINKED ISSUE #39: Code Exchange quality checklist
The purpose of this checklist is to assess the state of this repository, and bring it to an acceptable level for long-term support by completing the tasks listed below.

**This checklist is used internally but can be used as a guidance for others**

> Please check every part of this checklist for us to process your request. You can do this by marking them with [x] instead of [ ].

### Documentation

- [x] `LICENSE` file exists in repository

- [x] [Code of Conduct](https://github.com/twilio-labs/.github/blob/master/CODE_OF_CONDUCT.md) exists in repository

- [x] [CONTRIBUTING](https://github.com/twilio-labs/.github/blob/master/CONTRIBUTING.md) exists in repository 

- [x] No broken links in README.md

- [x] Build status exists at top of README.md

- [x] Set up steps are complete and accurate

- [x] CodeExchange repository is linked in README.md

### GitHub

- [x] Dependabot set up

- [x] Dependabot automerge set up

- [x] Uses GitHub Actions (required for Twilio sample) or other CI

### Code

- [x] Packages/dependencies are up to date

- [x] Code compiles against latest packages/dependencies

- [x] Matches [CodeExchange Guidelines](https://github.com/twilio-labs/code-exchange/tree/master/guidelines) to the best of ability

### Tests

- [x] Functionality tests have been created

- [x] All tests pass

### App

- [x] Confirmed sample behaves as expected (after dependencies have been updated)

- [x] This file has been copied into a wiki entry for the respective repository or added as a GitHub issue

### General

- [x] App is released under a permissive license like MIT or Apache-2

- [x] I've read and followed the contribution guidelines

- [x] I've read and implemented my template according to the respective coding guidelines for my project

- [x] I agree and adhere to the Code of Conduct

- [x] The template has been authored by me / or my organization and will be kept up-to-date. If the template will not longer be maintained, I'll notify this project.

### PR DESCRIPTION
**Pull request by rojasTob**
Resolves #39

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,10 +14,27 @@ This small sample application will say a short message to an inbound caller and,
...
 
-## Local Development
+| Config Value  | Description |
+| :-------------  |:------------- |
+TWILIO_ACCOUNT_SID / TWILIO_AUTH_TOKEN | In [Twilio Account Settings](https://www.twilio.com/console).
 
-This project is built using the [Flask](http://flask.pocoo.org/) web framework, Python 3, and the [Twilio Python Helper Library](https://www.twilio.com/docs/libraries/python).
+### Local development
 
 1. First clone this repository and `cd` into it.
```
Comment (bharatmurali) [inline README.md:39]: fix the numbering please :)

File: README.md [hunk 2]
```diff
@@ -74,10 +94,20 @@ This project is built using the [Flask](http://flask.pocoo.org/) web framework,
   https://<your-ngrok-subdomain>.ngrok.io/answer
```
Comment (bharatmurali) [inline README.md:94]: the lines above this need to be updated. the phone-numbers URL is old and needs to be updated to `https://www.twilio.com/console/phone-numbers/incoming`

File: README.md [hunk 3]
```diff
@@ -74,10 +94,20 @@ This project is built using the [Flask](http://flask.pocoo.org/) web framework,
...
   ```
 
+That's it!
```
Comment (bharatmurali) [inline README.md:97]: add one more step to say "Call your Twilio number to test it out" or something to that effect

**Comment by rojasTob** [comment]
Hi @bharatmurali, thanks for your review. The changes are done. :)

**Pull request merged**